app/views/trucks/_form.html.slim

```diff
@@ -10,7 +11,7 @@
   p
     = f.label :model_type
     br
-    = f.collection_select :model_type_id, ModelType.all, :id, :name
+    = f.collection_select :model_type_id, ModelType.all, :id, :name, {include_blank: 'Select a model'}
```

Comment:
(unless im mistaken because its .slim)
If the last argument is a hash in a Ruby method call, you can short-hand it by leaving out the curly braces.
Try using
`= f.collection_select :model_type_id, ModelType.all, :id, :name, include_blank: 'Select a model'`